Updates for 1.12 (#202)
informatici/openhospital-doc: doc_admin/AdminManual.adoc

include::https://raw.githubusercontent.com/informatici/openhospital/master/oh-bundle/OH-README.md[lines=25..413,leveloffset=+2]

=== 2.3 OH - SERVER mode (portable)

> Open Hospital can be lauched in SERVER (portable) mode. SERVER mode allows to launch a local instance of the database server and listen for connections from other clients.

Reviewer: ```suggestion
Open Hospital can be launched in SERVER (portable) mode. SERVER mode launches a local instance of the database server and listens for connections from other clients.
```

Author: Thank you David, fixed, sorry I wrote this quite in a rush for release, it will need some work in the future.

> It is useful to test/simulate a client/server configuration or networked environment; it is not suggested for production setup.

Reviewer: ```suggestion
This mode is useful to test/simulate a client/server configuration or networked environment; it is not suggested for production setup.
```

Author: Thank you David, fixed.

The custom logo has to be a 100x100 pixel png image file.

=== 2.2 OH - SERVER mode (portable)

> Starting with version 1.12, Open Hospital can be lauched in SERVER mode. SERVER mode allows to launch a local instance of the database server and listen for connections from other clients.

Reviewer: I know I'm a broken record but didn't we agree not to use "Starting with version xxx" in the text as the document always matches the version of the code?

Author: Yes,right, also reworked a little bit, thank you !

[source,properties]
----
###########################################
# This file contains Open Hospital settings
###########################################
# external settings
MODE=OH_MODE
###########################################
# internal settings, modules, directories
LANGUAGE=OH_LANGUAGE
SINGLEUSER=YES_OR_NO
DEBUG=no
DOC_DIR=OH_DOC_DIR
PATIENTPHOTOSTORAGE=PHOTO_DIR
INTERNALVIEWER=yes
SMSENABLED=no
VIDEOMODULEENABLED=yes
XMPPMODULEENABLED=no
###########################################
# application settings
ENHANCEDSEARCH=no
INTERNALPHARMACIES=yes
LABEXTENDED=yes
LABMULTIPLEINSERT=yes
MATERNITYRESTARTINJUNE=no
MERGEFUNCTION=yes
OPDEXTENDED=yes
PATIENTEXTENDED=yes
PATIENTVACCINEEXTENDED=yes

> ###########################################

Reviewer 2: I would replace ################## with just empty lines

Author: I think it's clearer in this way, we are using this standard also in makefile and scripts

Reviewer 2: makefile and scripts are not meant to be opened by users

parts like these ones result very confused and packed (like double title)
```
###########################################
# This file contains Open Hospital settings
###########################################
# external settings
MODE=OH_MODE
###########################################
# internal settings, modules, directories
LANGUAGE=OH_LANGUAGE
SINGLEUSER=yes
SINGLEUSER=YES_OR_NO
DEBUG=no
```

much more "bright" this way instead (less is more, spare characters, save the planet :) )

```
# This file contains Open Hospital settings

# external settings
MODE=OH_MODE

# internal settings, modules, directories
LANGUAGE=OH_LANGUAGE
SINGLEUSER=yes
SINGLEUSER=YES_OR_NO
DEBUG=no
```

[source,properties]
----
###########################################
# This file contains Open Hospital settings
###########################################
# external settings
MODE=OH_MODE

> ###########################################

Reviewer 2: To reflect new one in gui https://github.com/informatici/openhospital-gui/pull/1682/files#diff-1a8920a6e31a618e733a43e60b8e3da5c79b300b1756afaeab07ee953abc0ec5

Author: Fixed.

[cols=",,",options="header",]
|===
|Key |Default Value | Allowed Values

> |LANGUAGE |en |ar, de, en, es, fr, it, fr, pt, sw, zh_CN

Reviewer 2: ```suggestion
|LANGUAGE |en |ar, de, en, es, fr, it, pt, sw, zh_CN
```

Author: Fixed, thanks !

NOTE: An application restart is required to apply the modified setting.

==== 3.1.41 EXAMINATIONCHART
The following table shows the default value and the allowed ones:

[cols=",,",options="header",]
|===
|Key |Default Value | Allowed Values
|EXAMINATIONCHART |patient_examination | any kind of .jasper file name
|===

> Open Hospital can produce a report about the Patient Examinations.

Reviewer: ```suggestion
Open Hospital can produce a report about patient examinations.
```

Author: Yes, right, thank you !
